src/components/balances.vue

```diff
@@ -99,12 +99,26 @@
             tableData.value = {
                 data: balances.value.map(balance => {
                     return {
-                        balance: balance.balance,
+                        balance: balance.balance.toLocaleString(
+                            undefined,
```

Comment:
what is the behavior of undefined?

Reply:
I believe it's in place of a locale, however since we're formatting a number with only the option in mind we use undefined. This will let us limit the decimal places whilst including commas in the balance e.g. 123,456.789